### DISCUSSION TITLE
Ticket exchange/transfer

### DISCUSSION THREAD

**Discussion by simonsan** [pretix/pretix#3778] [Q&A]
> As a ticket buyer, I want to be able to resell my ticket, so that I can get back my money and someone else is able to get a ticket, that needs it.

> As an event organizer, I want the names on the ticket to be in line with the ownership of the ticket, so that it is easy to check the tickets with pretixScan on the entry. 

Could there be something done about it in `pretix` or is it advised to use external platforms? I would think `pretix` would be nice to use here, as it's already the selling platform. So people might be able to login, grab their ticket and transfer it to a person that they could invite to initiate a ticket transfer or something?

**Comment by raphaelm** [comment]
This should be a plugin. Others have tried before, but it is hard to solve it for the general case.
